<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>labelprint_goodinfo_dialog</class>
 <widget class="QWidget" name="labelprint_goodinfo_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="lds_toolbar_button_nowrap" name="pushButton_goodadd">
       <property name="text">
        <string>添加</string>
       </property>
       <property name="icon">
        <iconset resource="peripheral.qrc">
         <normaloff>:/image/add.png</normaloff>:/image/add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button_nowrap" name="pushButton_goodremove">
       <property name="text">
        <string>移除</string>
       </property>
       <property name="icon">
        <iconset resource="peripheral.qrc">
         <normaloff>:/image/printremove.png</normaloff>:/image/printremove.png</iconset>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button_nowrap" name="pushButton_print">
       <property name="text">
        <string>打印</string>
       </property>
       <property name="icon">
        <iconset resource="peripheral.qrc">
         <normaloff>:/image/printset.png</normaloff>:/image/printset.png</iconset>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="lds_toolbar_button_nowrap" name="pushButton_print_project_self">
       <property name="text">
        <string>自定义打印项</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_switch_mode">
       <property name="text">
        <string>正常模式(点击切换)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_allcontrol">
       <property name="text">
        <string>批处理</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_clearsection">
       <property name="text">
        <string>清除所选</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_toolbar_button</class>
   <extends>QPushButton</extends>
   <header>lds_toolbar_button.h</header>
  </customwidget>
  <customwidget>
   <class>lds_toolbar_button_nowrap</class>
   <extends>QPushButton</extends>
   <header>lds_toolbar_button_nowrap.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="peripheral.qrc"/>
 </resources>
 <connections/>
</ui>
